### DISCUSSION TITLE
Bosch unit in Smart Fortwo eq

### DISCUSSION THREAD

**Discussion by Jtiago99** [cedricp/ddt4all#871] [Q&A]
Hello,
I used to have a smart with r-link and successfully retrofitted reverse camera.

However now i have a 2021 with a Bosch unit and I can’t find the media system on ddt4all to activate the reverse camera, could someone help me finding that?

**Comment by KarelSvo** [comment]
The 52R0 radio is not built in Renault, but in Suzuki.  https://oss.bosch-cm.com/suzuki.html

**Comment by Jtiago99** [comment]
Thank you 🙏

It has a Renault part number 259150408R,

Do you know how can I code it? With which software?

Really appreciate

Tiago Melo

No dia 23/07/2023, às 09:03, KarelSvo ***@***.***> escreveu:

﻿

The 52R0 radio is not built in Renault, but in Suzuki. https://oss.bosch-cm.com/suzuki.html

**Reply by matzel82** [reply]
Yes....not a Renault Radio...

**Comment by Jtiago99** [comment]
Do you have any ideia how to code it?

I would pay for that information if necessary

Thank you

No dia 23/07/2023, às 10:01, matzel82 ***@***.***> escreveu:

﻿

Yes....not a Renault Radio...

**Reply by matzel82** [reply]
At the original garage... the best way to get it to work. Possible it's without software

**Comment by KarelSvo** [comment]
> Do you have any ideia how to code it?

You can try to connect to the A-IVI file and test if the data read is plausible.

**Comment by jacopogodeassi** [comment]
Hi, I'm Italian. I'm using a translator. Have you figured out how to access this radio to enable the rear camera? Thank you

**Comment by Jtiago99** [comment]
Hello,
No i didnt but my dealer activated for me.
Theo Charged me 50€

No dia 27/08/2024, às 20:38, jacopogodeassi ***@***.***> escreveu:

﻿

Hi, I'm Italian. I'm using a translator. Have you figured out how to access this radio to enable the rear camera? Thank you

**Comment by jacopogodeassi** [comment]
Hi, thanks for the answer. Did you manage to activate the rear camera on a smart eq with the latest navigator (Media System Connect) at the Mercedes dealership? Sorry again but I didn't understand well. Thank you

**Comment by Jtiago99** [comment]
Yes correct
Cumprimentos
Tiago Melo

No dia 27/08/2024, às 21:14, jacopogodeassi ***@***.***> escreveu:

﻿

Hi, thanks for the answer. Did you manage to activate the rear camera on a smart eq with the latest navigator (Media System Connect) at the Mercedes dealership? Sorry again but I didn't understand well. Thank you
